mongodb/mongodb_installer.sh

```diff
@@ -434,38 +533,48 @@ echo
 
 CONFIG_FILE="/usr/local/copperegg/ucm-metrics/mongodb/config.yml"
 AGENT_FILE="/usr/local/copperegg/ucm-metrics/mongodb/mongodb.rb"
+CONFIG_TEMPLATE="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/config-template.yml"
```

Comment:
No we have hardcoded the destination of config file, can't do the same for config template